src/chesster.js

```diff
@@ -422,8 +422,7 @@ chesster.controller.hears([
 	'direct_mention', 
 	'direct_message'
 ], function(bot,message) {
-    bot_exception_handler(bot, message, function(){
-    });
+    bot_exception_handler(bot, message, _.noop) 
```

Comment:
oh i like this. such simply things that I tend to do for myself. I never thought to look for them in a utility.